config/jobs/etcd/etcd-website-presubmits.yaml

```diff
@@ -0,0 +1,30 @@
+---
+presubmits:
+  etcd-io/website:
+  - name: pull-website-lint
+    cluster: eks-prow-build-cluster
+    optional: true # remove this once the job is green
```

Comment:
it's time to remove this?

Reply:
We usually remove this later after a new job has been introduced for a while and is behaving nicely in testgrid.

Reply:
OK